src/ripple/rpc/impl/KeypairForSignature.h

```diff
@@ -33,6 +34,7 @@ struct KeyPair
 };
 
 KeyPair keypairForSignature (Json::Value const& params, Json::Value& error);
+KeyPair keypairForSignature (KeyType keyType, RippleAddress const& seed);
```

Comment:
Please move these declarations can be moved to an already existing header, there's not enough here to justify an entire header file. The same thinking applies to this file:
https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/Sign.h